src/renderer/actions/utils/kill.js

```diff
@@ -1,23 +1,24 @@
-
+const { spawn } = require("child_process");
```

Comment:
Not using import syntax and incorrect brace-spacing for lint.

Reply:
Will do. The reason I didn't use your linter is because the code base was already full of linting errors and inconsistent styles, so it generated a ton of changes. I figured I wouldn't touch the linter for that reason.